apps/backend/src/app/interface/user.interface.ts

```diff
@@ -0,0 +1,14 @@
+import { Request } from 'express';
+import { User } from '../user/user.entity';
+
+export interface ICreateUser {
+  firstName: string;
+  lastName: string;
+  email: string;
+  firebaseId: string;
+}
+
+export interface ExtendedRequest extends Request {
+  user: User | null;
+  firebasePayload: any;
```

Comment:
Maybe there is a special type for that in firebase package, or you can create smth simple here. Any is not very good